french_law/python/src/allocations_familiales.py

```diff
@@ -1,59 +1,58 @@
 # This file has been generated by the Catala compiler, do not edit!
```

Comment:
Did you format this with `autopep8` ?

Reply:
Nope, I just did, thanks!

(BTW, any reason for choosing autopep8 over black?)

Reply:
I didn't know black :)